docs/api/browser-window.md

```diff
@@ -1198,6 +1198,12 @@ Returns `BrowserWindow` - The parent window.
 
 Returns `BrowserWindow[]` - All child windows.
 
+#### `win.setCursorAutoHide(autoHide)` _macOS_
```

Comment:
I think `setAutoHideCursor` would be a better name. Then it matches the existing `setAutoHideMenuBar` API and `disableAutoHideCursor` option.